src/components/NewDesign/Modals/SelectModals/SelectTokenModal/SelectTokenModal.tsx

```diff
@@ -27,21 +26,14 @@ export const SelectTokenModal: React.FC<ISelectTokenModal> = ({
   const classes = useStyles()
   const [value, setValue] = useState<string>('')
 
-  const descrpitionForSymbol: { [key: string]: string } = {
-    SOL: 'Solana',
-    USDC: 'USD Coin',
-    USDT: 'Tether',
-    BTC: 'Bitcoin',
-    USD: 'Dollar',
-    FTT: 'FTX Token',
-    ETH: 'Ethereum'
-
-  }
-
   const searchToken = (e: React.ChangeEvent<HTMLInputElement>) => {
     setValue(e.target.value)
   }
 
+  const tokenIndex = (name: string) => {
```

Comment:
what does this function do?

Reply:
pass token symbol to swap component and it is used for check token index in tokens array. (I removed mocked tokens list from modal)